src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -941,6 +939,9 @@ private static String getFirstDomainsFromIdent(
         /** Import line number. */
         private final int lineNumber;
 
+        /** Import end line number. */
+        private final int endLineNumber;
```

Comment:
Please rename `lineNumber` to `startLineNumber`.
Please add note in javadoc when it can be different values with startLineNumber

Reply:
Done! As well as getters.